### DISCUSSION TITLE
Group / Sort / Order Websites

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#286] [Anything Else]
Feedback on the new site switcher:

"Any chance we could get this list sorted (alphabetically maybe...)? I have 6 domains with a mix of subdomains, and they seem to be sorted by the order in which I added them. Would also be cool if subdomains were nested under their main domain."

[Via Twitter](https://twitter.com/RenegadeCoder94/status/1295226167201599488)

"i have quite a few websites and i would like to change my landing page. i would like to order it such that the important sites are on top and the open source projects are on the bottom. currently there does not seem to be a UI element for this."

([imported from the old roadmap](https://plausible.nolt.io/59))

"When you manage many websites, it would be great to be able to group them in a kind of folder maner or to tag them and filter them by tag (goupîng / tagging by server, by client, by kind, by host...)"

(from #411)

**Comment by Chaiavi** [comment]
I have about 100 sites, grouping is a must for me

Please upvote

**Comment by RobertJoonas** [comment]
Another vote for this:

> On the /sites page it would be nice if I could assign each website to a group, in my case Departments, so that it’s easier to find a site.
> Like there would be a group ‘Marketing’, a group ‘Operations’, a group ‘Sales’ … with each their own websites.

(feedback via email)

**Comment by LauGau** [comment]
YES YES YES :)
Maybe tags so if we need one website can be displayed in multiple categories.
Ex: "#ServerX" AND "#ClientY".

But IMO on this screen it's even more important to be able to see Visitors count for a month and not the last 24h (not relevant for low traffic websites.) Discussion here : https://github.com/plausible/analytics/discussions/1035

**Comment by GarveScott-Lodge** [comment]
The ability to tag or categorise sites and then filter them on the My Sites page would be very useful to me too. In my case it would be to divide those clients who were paying me for their access to Plausible and those who I have given access for a trial period, but I can see lots of possible uses.

**Comment by shyjal** [comment]
Categorising is much needed

**Comment by rdvo** [comment]
Yea +1 for this, would be very nice to sort sites.. the product is good, it will grow and continue to. The fact this is a need is a testament to people using it for their agencies and/or client sites.. ultimately growth for the software

**Comment by gbissland** [comment]
+1 please. We have a large amount of sites now on Plausible and it is a horrible experience to find and navigate to some of them via the dashboard.

**Comment by bogplau** [comment]
There's a workaround that will help until we have a better system in place. 

If you have a large amount of sites added to your account, you can bypass the default 18 website/page by appending `per_page=xxx ` to display more websites. 

Example: `https://plausible.io/sites?per_page=100`

Then, simply hit ctrl/cmd+F on your keyboard to search for a particular website.

**Comment by marcbria** [comment]
+1 to this FR.

For services that grow, categorization and listing is a must.
All sorting criteria suggested sound good but "most visited" sorting would be my first choice.

**Comment by ZapaNOR** [comment]
+1 For grouping / folders / organisations.
We have a LOT of sites on plausible for our clients, and its getting messy trying to get an overview.

**Comment by GarveScott-Lodge** [comment]
Bumping this again. I now have ~40 sites and could really do with being able to group/filter them.

**Comment by gbissland** [comment]
The new UI refresh for the websites grid is nice.

It looks like it has broken the pagination 'hack' to be able to view more sites per page though.
i.e https://plausible.io/sites?per_page=xxx

Any update on any grouping or filtering options for this screen?

**Reply by metmarkosaric** [reply]
Thanks @gbissland, glad you like the new improvements! That pagination was a hack that we used before we improved the sites page to introduce search. Now with the search functionality being available it’s easy to find the site you’re looking for even without the hack. We’re currently working on the ability to pin sites so you’ll be able to choose those sites you want to show up on the top of the sites list. Stay tuned!

**Comment by metmarkosaric** [comment]
**Update**: You can now pin sites that you want to feature at the top of your sites page. You can pin up to nine sites. The most recently pinned site will automatically appear at the top of your list, helping you organize your dashboards by how often you use them.

Your pinned sites will also show up in the site switcher list that you can see within each dashboard by clicking on the site name in the top left. You can navigate between the sites in the site switcher list using keyboard shortcuts 1 to 9.

**Comment by Chaiavi** [comment]
Thank you Marko.

I still want grouping and feel it is needed but Thanks for the new feature,
it is helpful!

**Comment by LauGau** [comment]
I agree, with @Chaiavi, grouping would be nice. Also we can "pin" inside group, it should not be exclusive. So people can create groups... or not.

**Comment by gbissland** [comment]
Agreed, thanks for this pinning!!.... but this request should stay open as further Grouping for us is still preferred.

**Comment by mgussekloo** [comment]
It's been a while since the last comment here, but I think the sites dashboard still could use some work. Page navigation should also be on the top, and be more clear about the current results / total results (something like "Showing ... of ... results"). It would be nice if it could be sorted by name, date of creation, number of visitors, etc. I'd like it to be visually clear when a site is pinned to the top.

I hope this is the right place for this feedback!
